lib/specifications/Specification.js

```diff
@@ -151,6 +143,17 @@ class Specification {
 		return this._specVersion;
 	}
 
+	/**
+	 * Returns an instance of a helper class allowing for convenient comparison
+	 * operations against this instance's Specification Version
```

Comment:
Does everybody know what a Specification Version is? I'm wondering if this whole versioning business is explained somewhere in the documentation.

Reply:
We have a chapter in our documentation regarding that: https://sap.github.io/ui5-tooling/pages/Configuration/#specification-versions